src/App.tsx

```diff
@@ -18,186 +172,91 @@ export const App: React.FC = () => {
           {/* this button should have `active` class only if all todos are completed */}
           <button
             type="button"
-            className="todoapp__toggle-all active"
+            className={`todoapp__toggle-all ${completedCount === todos.length ? 'active' : ''}`}
```

Comment:
Use the classnames library for add classes with condition and move this logic from header tag to the separate `Header` component